contributors/triage/README.md

```diff
@@ -0,0 +1,13 @@
+# Kubernetes Issue Triage
+
+This directory contains Kubernetes Issue Triage Guidelines and related documents.
+
+Triaging issues is a great way to get started contributing, and it will help you understand how the project is organized. Following are few predetermined searches for convenience:
```

Comment:
There are 100 new issues created every week.

I think the most important thing to do is to figure out how to scale the triage process.